### DISCUSSION TITLE
📅 Suggest Meeting Times for the Web-Dev Team

### DISCUSSION THREAD

**Discussion by yigakpoa** [clentafrica/web-dev#13] [Polls]
Hey team @clentafrica/wg-web-dev 

We’re planning to schedule our next web-dev team meeting and would love your input on the best time for everyone to join! Please select your preferred time slots in the poll below. We’re looking for a time that works for most of us from now until Friday.

### Available Time Slots:
- **Wednesday**: 
  - 10:00 AM UTC
  - 2:00 PM UTC
  - 5:00 PM UTC

- **Thursday**: 
  - 9:00 AM UTC
  - 1:00 PM UTC
  - 4:00 PM UTC

- **Friday**: 
  - 11:00 AM UTC
  - 3:00 PM UTC
  - 6:00 PM UTC

Feel free to vote for multiple options if you’re flexible, and if none of these work, please comment below with your availability. We'll go with the most popular time that suits the majority.

I hope we find a time that works for all of us! Thanks, everyone, and looking forward to connecting soon!

**Comment by uchiha-vivek** [comment]
@yigakpoa after the poll please confirm the final time as well for web dev team .
Thank you 😊

**Reply by yigakpoa** [reply]
Hi @uchiha-vivek its in 30 minutes

**Reply by yigakpoa** [reply]
Hi @clentafrica/wg-web-dev, please find the link to the meeting in 30 minutes: https://us05web.zoom.us/j/84230103095?pwd=vcHmQf8ezxrctZnHsyglnV10PADebJ.1

**Reply by uchiha-vivek** [reply]
@yigakpoa ready

**Reply by yigakpoa** [reply]
It’s @clentafrica/wg-web-dev it’s time

**Comment by yigakpoa** [comment]
Hi all please add your emails here or send on discord to be added to the backend of the site.

-

**Reply by MadhalasaSJ** [reply]
madhalasasj1407@gmail.com

**Reply by uchiha-vivek** [reply]
@yigakpoa viveksharma7497@gmail.com

**Comment by adilkadivala** [comment]
adilkadivala560@gmail.com

**Reply by adilkadivala** [reply]
hey @yigakpoa
